### ISSUE TITLE
Add support for JSX-Fragments

### ISSUE THREAD

**Issue by marvinhagemeister** [prettier/prettier#3341]
**Prettier 1.8.2**
[Playground link](https://prettier.io/playground/#N4Igxg9gdgLgprEAuc0DOMAEAxCFMC8mAPAHwA6Um1N1xAJgJYBupAEnADaf4DqEAJ071iAeiatKY0gG5KIADQgIABxiN0yUAEMMyAGbbOaOEoBGA7WADWcGAGUVVxlADmyGAICupkPQhgBkYmSi4mAjAACpauALbaQca+AFZoAB4AQpY2dvbasXAAMi5wiSHKXjAqlQBMZb5OAuHIIGbaZgCePFCKICoCLjC8jPQwABbIABwADEr9ECa8lioe3r4CcACOXowb0dpxCUiGSUomsYyrPmcurpxwAIpeEPD1SjDtw6MTSDXvloxOLcAMIQWLxFpQaClJReEwAFXaaHqAF8UUA)

**Input:**
```jsx
const Foo = <>
        <div>Hello World</div>
</>;

```

**Output:**
```jsx
SyntaxError: Unexpected token (1:14)
> 1 | const Foo = <>
    |              ^
  2 |         <div>Hello World</div>
  3 | </>;
  4 | 
```

**Expected behavior:**

React `16.2.0` was  just released with support for jsx fragments. These are currently not recognized by prettier and lead to syntax error.

Link to [JSX-Fragments spec](https://github.com/facebook/jsx/blob/master/AST.md#jsx-fragment)

**Comment by ikatyang** [comment]
There's already a WIP PR, see #3237.

**Comment by ikatyang** [comment]
Fixed in #3237.

**Comment by marvinhagemeister** [comment]
@ikatyang That's awesome! Much appreciated 👍
